bin/config.py

```diff
@@ -0,0 +1,6 @@
+"""Configuration file for server.py"""
```

Comment:
It would be great if you added a title like this:https://github.com/tolstoyevsky/shirow/blob/master/shirow/server.py. And in the server.py too.